tests/Maui/Prism.DryIoc.Maui.Tests/Fixtures/Navigation/NavigationTests.cs

```diff
@@ -117,6 +117,38 @@ public async Task AddsModalPageFromRelativeURI()
         Assert.IsType<MockViewB>(rootPage.Navigation.ModalStack.Last());
     }
 
+    [Fact]
+    public async Task FlyoutRelativeNavigation_RemovesPage_AndNavigatesNotModally()
+    {
+        var mauiApp = CreateBuilder(prism => prism.OnAppStart("MockHome/NavigationPage/MockViewA"))
+            .Build();
+        var window = GetWindow(mauiApp);
+
+        var rootPage = window.Page as MockHome;
+        Assert.NotNull(rootPage);
+        TestPage(rootPage);
+        Assert.NotNull(rootPage.Detail);
+        var detailPage = rootPage.Detail as NavigationPage;
+        Assert.NotNull(detailPage);
+        TestPage(detailPage);
+        var currentPage = detailPage.CurrentPage;
+        Assert.IsType<MockViewA>(currentPage);
+        TestPage(currentPage);
+        var container = rootPage.GetContainerProvider();
+        var navService = container.Resolve<INavigationService>();
```

Comment:
```suggestion
        var navService = rootPage.GetNavigationService();
```

Reply:
I found only one `GetNavigationService` method, but it's not an extension method. I hope it's fine now.

Reply:
you are correct I forgot I did not make that one an extension method.